src/lib/models/event-history/get-event-categorization.test.ts

```diff
@@ -64,6 +64,7 @@ describe('Event Category Data Structures', () => {
         "WorkflowExecutionTerminated": "workflow",
         "WorkflowExecutionTimedOut": "workflow",
         "WorkflowExecutionUpdateAccepted": "update",
+        "WorkflowExecutionUpdateAdmitted": "update",
```

Comment:
We might want to add this to `const categories` here ⬇️ in this test file as well

```js
  update: [
    'WorkflowExecutionUpdateAccepted',
    'WorkflowExecutionUpdateAdmitted',
    'WorkflowExecutionUpdateCompleted',
  ],
```

Reply:
👍